endpoints/carparks/index.js

```diff
@@ -1,53 +1,57 @@
-var request = require('request')
-var $ = require('cheerio')
-var h = require('apis-helpers')
-var app = require('../../server')
+import request from 'request'
+import $ from 'cheerio'
+import h from 'apis-helpers'
+import app from '../../server'
 
-app.get('/carparks', function (req, res) {
-
-  var url = 'http://www.bilastaedasjodur.is/'
+app.get('/carparks', (req, res) => {
+  const url = 'http://www.bilastaedasjodur.is/'
 
   request.get({
     headers: { 'User-Agent': h.browser() },
-    url: url
-  }, function (error, response, body) {
-    if (error || response.statusCode !== 200)
-      return res.status(500).json({ error:'www.bilastaedasjodur.is refuses to respond or give back data' })
+    url,
+  }, (error, response, body) => {
+    if (error || response.statusCode !== 200) {
+      return res.status(500).json({
+        error: 'www.bilastaedasjodur.is refuses to respond or give back data',
+      })
+    }
 
     try {
-      var data = $(body)
-    } catch (error) {
-      return res.status(500).json({ error:'Could not parse body' })
+      // FIXME: What is the point of this? It doesn't seem to be used anywhere
+      // eslint-disable-next-line no-unused-vars
+      const data = $(body)
```

Comment:
@haraldurkarls I stumbled upon this while updating this file to be lint-compliant

What is the point of this? It doesn't seem to be used anywhere